tests_pycsob/test_api.py

```diff
@@ -96,14 +99,16 @@ def test_payment_init_success(self):
             ('resultMessage', 'OK'),
             ('paymentStatus', 1)
         ))
-        responses.add(responses.POST, resp_url, body=json.dumps(resp_payload), status=200)
+        responses.add(responses.POST, utils.mk_url(BASE_URL, '/payment/init'), body=json.dumps(resp_payload),
+                      status=200)
         out = self.c.payment_init(order_no=666, total_amount='66600', return_url='http://example.com',
                                   description='Nějaký popis').payload
 
         assert out['paymentStatus'] == conf.PAYMENT_STATUS_INIT
         assert out['resultCode'] == conf.RETURN_CODE_OK
         assert len(responses.calls) == 1
 
+    @freeze_time()
```

Comment:
This actually solves the test problem:

- `utils.dttm()` basically return specially formatted result of `datetime.now()`
- since the result of the above function is time-dependent and is a part of the URL used in `responses`, time may tick between the creation of the mock and actually calling the endpoint - resulting in `responses` complaining that
`requests.exceptions.ConnectionError: Connection refused by Responses - the call doesn't match any registered mock.` and failing the test.

Get rid of the test problem by freezing time.